shift_detector/detector.py

```diff
@@ -73,6 +81,16 @@ def run(self, *checks, logger_level=logger.ERROR):
                                       shifted_columns=[],
                                       information=error_msg)
                 check_reports.append(error_report)
+
+        if download_nltk:
```

Comment:
I wonder, whether this could actually work because the libraries are downloaded after the checks are executed.